packages/@lwc/ssr-compiler/src/generator.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import type { Instruction } from './shared';
+import type { Program } from 'estree';
+
+export function bytecodeToEsTree(_instructions: Generator<Instruction>): Program {
```

Comment:
I don't see any usage of this.

Reply:
Thanks! This was a vestigial stub from early on in the spike. Removed.